### PR TITLE
update(options/datadirs): add v6 use-case specifics to options page

### PR DESCRIPTION
**Pull request by convexshiba**
specify situations where `dataDirs` should be used.

### CODE REVIEW

**Comment by zakkarry** [comment]
I think that this is going to need to be restructured and formatted in a more presentable way then one big long block of text like that.

**Comment by convexshiba** [comment]
Sure. Tried to simplify the logic better, to the best of my current understanding.

File: docs/basics/options.md [hunk 1]
```diff
@@ -439,6 +439,14 @@ above. Back-slashes are "escape characters" and "\\" equates to "\"
...
 where it cannot find a associated torrent file.
 
+When using V6, this field should only be set in two specific use-cases: 
+1. if you are downloading through usenet or other non-torrent means. This will always be necessary if 
+you want cross-seed to match against this content. 
+2. if you have content in your media directories not inside your torrent client. Here you only need
+to perform a search with `dataDirs` ONCE. At the moment, the best way to do this is to add the relevant
+directories to your `dataDirs` setting and then delete them later. This use-case could be better supported in a future update.
+
+
```
Comment (zakkarry) [inline docs/basics/options.md:449]:
```suggestion
#### General Usage in v6

Starting in v6.0.0, this option is generally only applicable in two cases:

1. You are downloading through Usenet or other non-torrent methods in order to
   match new content not present in your torrent client.

2. You have content in your media or data directories that is not already
   present in your torrent client. In this scenario, you only need to perform a
   search with `dataDirs` **once**. After the initial search, you should remove
   the directories from `dataDirs` entirely.
```

**Review by zakkarry — CHANGES_REQUESTED** [review]
Open to feedback on this rewrite/formatting.

Let me know what you think.

**Review by zakkarry — APPROVED** [review]

**Comment by zakkarry** [comment]
If I'm not mistaken this is just my changes exactly? You were good with them?

If you have any thoughts please feel free to let me know, otherwise, I can merge these after you give a final "OK"
<hr>
I think we may have regrettably not gotten off on the best of starts. I have some serious family/personal issues going on, and that may have (definitely) influenced some of my shortness with you yesterday. I was, however, genuinely trying to understand the confusion on #61 - as I felt it was rather clear and unambiguous.

The way we word stuff as developers sometimes, despite being "correct and clear" to us and maybe even "according to Webster", may not always be clear to the average user. This is more what I was attempting to evoke from you, an explanation of where we had fallen short and how; not as much taunting you with definitions and "ya but look here" type of stuff.

I can see, in hindsight, where it may have come across like that given the context and our previous interaction, etc.

I am very sorry for that. I have no justification. I hope you aren't too discouraged in continuing to be a part of this project and community at whatever level you find appropriate going forward.

I'll be on Discord if you'd like to speak further. Otherwise, give me a comment of "lgtm" (looks good to me) - and I will gladly merge.

Appreciate your having made the contribution and taking the time.

zak

**Comment by convexshiba** [comment]
Hey, yeah of course, your edits are more concise and integrate better with the overall section. Good to ship.

And you don't owe me any explanations but apologies accepted,. For a FOSS it's already above and beyond to offer live debugging help. I appreciate that. And my issues were also resolved pretty quickly, what else one would ask for.

I work in the software industry, so I know documentation oftentimes come down to personal style and preferences. So for #61, I can respect your, as dev's, opinion that a certain statement is sufficiently clear. That's why I think it's ok to close that one. But let me elaborate more in #61 what I was meaning.

**Comment by zakkarry** [comment]
> Hey, yeah of course, your edits are more concise and integrate better with the overall section. Good to ship.
> 
> And you don't owe me any explanations but apologies accepted,. For a FOSS it's already above and beyond to offer live debugging help. I appreciate that. And my issues were also resolved pretty quickly, what else one would ask for.
> 
> I work in the software industry, so I know documentation oftentimes come down to personal style and preferences. So for #61, I can respect your, as dev's, opinion that a certain statement is sufficiently clear. That's why I think it's ok to close that one. But let me elaborate more in #61 what I was meaning.

Thanks for understanding, sorry I was a bit abrasive.

**Pull request merged**